server/src/main/java/org/opensearch/search/profile/query/InternalProfileCollector.java

```diff
@@ -68,13 +68,47 @@ public class InternalProfileCollector implements Collector, InternalProfileCompo
     /** The wrapped collector */
     private final ProfileCollector collector;
 
+    /** The elapsed time in the concurrent search reduce phase */
+    private final Long reduceTime;
+
+    /** The maximum slice time for this collector manager */
+    private final Long maxSliceTime;
+
+    /** The minimum slice time for this collector manager */
+    private final Long minSliceTime;
+
+    /** The average slice time for this collector manager */
+    private final Long avgSliceTime;
+
+    /** The segment slice count for this collector manager */
+    private final int sliceCount;
+
     /**
      * A list of "embedded" children collectors
      */
     private final List<? extends InternalProfileComponent> children;
 
     public InternalProfileCollector(Collector collector, String reason, List<? extends InternalProfileComponent> children) {
         this.collector = new ProfileCollector(collector);
+        this.reduceTime = null;
```

Comment:
`Please call super(...)`

Reply:
Hi @reta, thank you for your comment.

Upon double-checking `InternalProfileCollector` class, I have confirmed that there is no superclass constructor for this class. Instead, `InternalProfileCollector` class implements `Collector` and `InternalProfileComponent` interfaces.

Reply:
My apologies, I meant 'this(...)` - to reuse the new constructor you've introduced

Reply:
Got it. It's addressed now.